src/main/plugin/iso19139.ca.HNAP/schematron/schematron-rules-non-multilingual.sch

```diff
@@ -72,8 +72,8 @@
     <!-- Metadata Standard Name -->
     <sch:rule context="//gmd:metadataStandardName">
 
-      <sch:let name="correct" value="(($mainLanguageId = 'eng' and gco:CharacterString = 'North American Profile of ISO 19115:2003 - Geographic information - Metadata') or
-                          ($mainLanguageId = 'fra' and gco:CharacterString = 'Profil nord-américain de la norme ISO 19115:2003 - Information géographique - Métadonnées'))
+      <sch:let name="correct" value="(($mainLanguage = 'eng' and gco:CharacterString = 'North American Profile of ISO 19115:2003 - Geographic information - Metadata') or
```

Comment:
Above `lower-case($mainLanguage)` is used for this test, do we need to be consistent for this to work?

Reply:
I would apply `lower-case` to the `select` expression in `$mainLanguage` to avoid having to lower case the value in each usage.

Like:

```
  <sch:let name="mainLanguage" value="if (normalize-space(//*[name(.)='gmd:MD_Metadata' or @gco:isoType='gmd:MD_Metadata']/gmd:language/gmd:LanguageCode/@codeListValue) != '')
                                       then lower-case(normalize-space(//*[name(.)='gmd:MD_Metadata' or @gco:isoType='gmd:MD_Metadata']/gmd:language/gmd:LanguageCode/@codeListValue))
                                       else if (contains(//*[name(.)='gmd:MD_Metadata' or @gco:isoType='gmd:MD_Metadata']/gmd:language/gco:CharacterString,';'))
                                            then lower-case(normalize-space(substring-before(//*[name(.)='gmd:MD_Metadata' or @gco:isoType='gmd:MD_Metadata']/gmd:language/gco:CharacterString,';')))
                                            else lower-case(//*[name(.)='gmd:MD_Metadata' or @gco:isoType='gmd:MD_Metadata']/gmd:language/gco:CharacterString)"/>
```

Reply:
@jodygarnett - Looks good.  Thanks you for applying the fix.